tidy3d/components/autograd/boxes.py

```diff
@@ -0,0 +1,53 @@
+# Adds some functionality to the autograd arraybox
+# NOTE: this is not a subclass of ArrayBox since that would break autograd's internal checks
+
+import autograd.numpy as anp
+from autograd.numpy.numpy_boxes import ArrayBox
+
+TidyArrayBox = ArrayBox  # NOT a subclass
```

Comment:
yeah more or less. it's somewhat hacky and modifies the underlying autograd arraybox too, although that doesn't really matter since we're not breaking anything. but yeah not ideal. an actual fork might still be something to consider...

Reply:
sounds good. we can reconsider this later